tests/core/query/test_relational_query.py

```diff
@@ -699,14 +699,15 @@ def test_labels_table_joins(full_sdata):
         table_name="table",
         how="left",
     )
-    assert all(table.obs["instance_id"] == range(100))
+
+    assert all(table.obs["instance_id"] == range(1, 100))
```

Comment:
here the labels has values 0,100 but the 0 element is background, so is effectively filtered in the left/right joins when `get_element_instances` is called on labels. Hence the indices of the instances are different.